.github/workflows/docs.yml

```diff
@@ -6,14 +6,14 @@ on:
       - master
       - develop
       - feature/github-actions
-      - 'feature/doc-**'
+      - feature/doc-**
     tags: '*'
   pull_request:
     branches:
       - master
       - develop
       - feature/github-actions
-      - 'feature/doc-**'
+      - feature/doc-**
```

Comment:
Maybe we could add a `doc/` category. It would not run tests or benchmarks but would still build docs. This can be done in a separate PR.